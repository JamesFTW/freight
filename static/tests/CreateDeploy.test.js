import React from 'react';
import renderer from 'react-test-renderer';
import CreateDeploy from '../components/CreateDeploy.jsx';

jest.mock('../components/CreateDeploy.jsx')
//TODO: figure out why snapshots are rendering null.
test('CreateDeploy Snapshot', () => {
  const component = renderer.create(
    <CreateDeploy />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
