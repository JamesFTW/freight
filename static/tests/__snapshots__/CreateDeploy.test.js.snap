// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateDeploy Snapshot 1`] = `null`;
